<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Room</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="msgBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>601</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendTxtMsg">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMore">
    <property name="title">
     <string>More</string>
    </property>
    <addaction name="info"/>
    <addaction name="logout"/>
   </widget>
   <addaction name="menuMore"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="info">
   <property name="text">
    <string>Personal information</string>
   </property>
  </action>
  <action name="logout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
